utility/src/main/java/org/oppia/android/util/parser/html/CustomHtmlContentHandler.kt

```diff
@@ -196,16 +196,22 @@ class CustomHtmlContentHandler private constructor(
      */
     fun <T> fromHtml(
       html: String,
-      imageRetriever: T,
+      imageRetriever: T?,
```

Comment:
I think it's fine to have a nullable property for a helper, but for the public API I suggest making things more explicit by having 1 version that takes a non-null image retriever and another which takes nothing but the HTML.

Reply:
Done